pkg/config/config_template.yaml

```diff
@@ -87,12 +87,11 @@ api_key:
 # Default is '5002' on Windows and macOS ; turned off on Linux
 # GUI_port: -1
 
-
 # The Agent runs workers in parallel to execute checks. By default the number
-# of workers is automatically determined based on the number of checks running.
-# This optimizes checks collection time but might produce CPU spikes. You can
-# enforce a fix number of workers so less CPU is used.
-# check_runners: 4
+# of workers is set to 1. If set to 0 the agent will automatically determined
+# the best number of runners needed based on the number of checks running. This
+# would optimize the check collection time but will produce CPU spikes.
```

Comment:
let's say a conservative `may produce CPU spikes`, what do you think?

Reply:
👍